platform/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -41,7 +41,7 @@ spec:
         - --parentShutdownDuration
         - 3s
         - --discoveryAddress
-        - istio-pilot:8080
```

Comment:
Why make this change throughout the whole PR?

Reply:
Upgrade becomes easier, we need separate ports for envoy in front of mixer and pilot.